extensions/amp-analytics/0.1/requests.js

```diff
@@ -30,7 +30,7 @@ import {dict, map} from '../../../src/utils/object';
 import {filterSplice} from '../../../src/utils/array';
 import {isArray, isFiniteNumber} from '../../../src/types';
 
-const TAG = 'AMP-ANALYTICS';
+const TAG = 'AMP-ANALYTICS/requests';
```

Comment:
Can we agree on the `TAG` format? Either is fine to me, but same format please.

Reply:
We have those in analytics code right now:

```
analytics-events
amp-analytics.IframeTransport
amp-analytics.Transport
amp-analytics.IframeTransportMessageQueue
AMP-ANALYTICS
VISIBILITY-MANAGER
amp-analytics/linker-manager
```

I suggest we go with 
```
<extensionName>/<fileName>
```

like:
```
amp-analytics/requests
```

Can we address it in a follow up PR to fix all in one shot?

Reply:
sg